src/nidcpower/examples/nidcpower_constant_resistance_and_constant_power.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+
+import argparse
+import nidcpower
+import sys
+
+
+def example(
+    resource_name,
+    options,
+    output_function,
+    constant_resistance_level,
+    constant_resistance_level_range,
+    constant_resistance_current_limit,
+    constant_power_level,
+    constant_power_level_range,
+    constant_power_current_limit,
+    source_delay,
+):
+    assert output_function in (
+        nidcpower.OutputFunction.CONSTANT_RESISTANCE, nidcpower.OutputFunction.CONSTANT_POWER
+    ), 'This example only supports CONSTANT_RESISTANCE and CONSTANT_POWER output functions.'
+
+    with nidcpower.Session(resource_name=resource_name, options=options) as session:
+        # Configure the session.
+        session.source_mode = nidcpower.SourceMode.SINGLE_POINT
+        session.output_function = output_function
+        if output_function == nidcpower.OutputFunction.CONSTANT_RESISTANCE:
+            session.constant_resistance_level = constant_resistance_level
+            session.constant_resistance_level_range = constant_resistance_level_range
+            session.constant_resistance_current_limit = constant_resistance_current_limit
+        else:
+            session.constant_power_level = constant_power_level
+            session.constant_power_level_range = constant_power_level_range
+            session.constant_power_current_limit = constant_power_current_limit
+        # Configure the source delay to allow for sufficient startup delay for the input to reach
+        # the desired sinking level. When starting from a 0 A or Off state, the electronic load
+        # requires additional startup delay before the input begins to sink the desired level. The
```

Comment:
Should this be "sink **to** the desired level"?

Reply:
I've updated it in the latest commit.